pyirf/interpolation/tests/test_moment_morph_interpolator.py

```diff
@@ -29,14 +29,14 @@ def bins():
 def simple_1D_data(bins):
     grid = np.array([[20], [40]])
     target = np.array([30])
-    bin_contents = np.array([binned_normal_pdf([x, 0], bins) for x in grid])
+    binned_pdf = np.array([binned_normal_pdf([x, 0], bins) for x in grid])
```

Comment:
`binned_normal_pdf` produces test-cases where the pdf sums to one, not integrates to one. As these cases here are all with a linear binning this global factor should not change anything, however we should probably change the test case to better reflect the use-case